fiftyone/operators/delegated.py

```
@@ -895,7 +898,14 @@ async def _execute_operator(self, doc):
 
             operator, _, ctx, __ = prepared
 
-            result = await do_execute_operator(operator, ctx, exhaust=True)
+            if doc.pipeline:
+                result = await do_execute_pipeline(doc.pipeline, ctx)
+                if result:
+                    error, error_message = result
```

Ruff (0.14.6)

Unpacked variable `error_message` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)